<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFormPancake3D</class>
 <widget class="QWidget" name="MainFormPancake3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3D Pancake</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_cs">
       <property name="text">
        <string>cs (lower = tighter fit)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_edit_c_s">
       <property name="text">
        <string>0.67</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_xy_scale">
       <property name="text">
        <string>XY Scale (nm):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_edit_xy_scale">
       <property name="text">
        <string>5.03</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_z_scale">
       <property name="text">
        <string>Z Scale (nm):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_edit_z_scale">
       <property name="text">
        <string>42.017</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6"/>
   </item>
   <item>
    <widget class="QCheckBox" name="chk_visualize_steps">
     <property name="text">
      <string>Visualize Steps</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chk_visualize_results">
     <property name="text">
      <string>Visualize Results</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="line_edit_file">
       <property name="placeholderText">
        <string>CSV File Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_file_select">
       <property name="toolTip">
        <string>Select Folder</string>
       </property>
       <property name="text">
        <string>Select Output Folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="line_edit_filepath">
     <property name="placeholderText">
      <string>Output Filepath</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_select_multiroi">
       <property name="text">
        <string>Select Multiple PSDs (MultiROI)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_select_roi">
       <property name="text">
        <string>Select Single PSD (ROI)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_selected">
     <property name="text">
      <string>Selected: None</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_process">
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CopyableLabel" name="label_output">
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CopyableLabel</class>
   <extends>QLineEdit</extends>
   <header>.copyable_label</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
